### ISSUE TITLE
Defaults not used

### ISSUE THREAD

**Issue by lettersfromfelix** [watr/mbeacon#4]
Hi, I just began working with mbeacon, so probably I just missed something. But whenever I set according to the readme the uuid or measuredPower values via defaults and start mbeacon afterwards, it seems to completely ignore them? At least the terminal output of the command always shows a random uuid and measuredPower = 0. When giving UUID as arguments to mbeacon, it shows the correct one (but it seems like I'm stuck with measuredPower=0). 
Could this be the reason why I can't detect any such beacon as well? Or is there another problem which I'm not aware of yet? 

Cheers,
Flacedoo

(MacOS 11.4 Big Sur, MacBook Pro 2016)

**Comment by watr** [comment]
@Flacedoo Thank you for the feedback.
Your point is correct.
Current version (v0.2.0) mbeacon can't find defaults value because identifier of binary (mbeacon) have changed implicitly. https://github.com/watr/mbeacon/commit/e409827f46dd7c85c4e7aed157cdbdaa3bea4b71
`mbeacon -> dev.prancy.mbeacon`
I'll revert identifier `dev.prancy.mbeacon -> mbeacon` and release new binary.

If you are in a hurry, use new identifier (dev.prancy.mbeacon) in command line.

```
$ defaults write dev.prancy.mbeacon uuid -string "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
$ defaults write dev.prancy.mbeacon measuredPower -integer -45
$ mbeacon -major 0 -minor 0
```

**Comment by watr** [comment]
Fixed with f1b41e2835c13331b4b6599640229ed94cdb8e77 .  I've released v2.0.1.

## How to use

```
$ defaults write mbeacon uuid -string "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
$ defaults write mbeacon measuredPower -integer -45
$ mbeacon -major 0 -minor 0
```

or 

```
$ defaults write mbeacon measuredPower -integer -45
$ mbeacon -uuid "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX" -major 0 -minor 0
```

**Comment by watr** [comment]
Tested v2.0.1 mbeacon binary in macOS 10.15.7
